tests/scalar/test_math.py

```diff
@@ -59,6 +61,14 @@ def test_gammaincc_nan_c():
     assert np.isnan(test_func(-1, -1))
 
 
+def test_gammaincinv_python():
```

Comment:
Usually we don't test the Scalar version unless it's something fancy. Here we're just calling scipy.

Instead we test the Elemwise version in pytensor.tensor which is usually what users see.